Final Project/Portfolio/css/styles.css

```diff
@@ -0,0 +1,207 @@
+/* 
+====== BRAND ====== 
+COLORS
+	blue #2C3E50
+	gray #ECF0F1
+	black #333333
+FONT
+	font-family: 'Raleway', Helvetica, Arial sans-serif;
+*/
+
+/* ====== GLOBAL ====== */
+html {
+	font-size: 16px;
+}
+* {
+	box-sizing: border-box;
+}
+body {
+	background-color: #ECF0F1;
+	font-family: 'Raleway', Helvetica, Arial sans-serif;
+}
+.content {
+	width: 70%;
+	margin: 0 auto;
+}
+
+/* ====== HEADER ====== */
+header {
+	background-color: #2C3E50;
+	width: 100%;
+	height: 200px;
+	color: #ECF0F1;
+	font-weight: 300;
+}
+.contact-header {
+	height: auto;
+}
+	header img {
+		width: 160px;
+		height: 160px;
+		border-radius: 50%;
+		margin: 25px 20px 0 0;
+		float: left;
+	}
+	h1 {
+		font-size: 2.25rem;
+		line-height: 1.3;
+		margin: 60px 0 0 0;
+		display: inline-block;
+	}
+	h1 a {
+		color: #ECF0F1;
+		text-decoration: none;
+	}
+	h1 a:hover {
+		text-decoration: underline;
+	}
+	.contact-h1 {
+		margin: 10px 0;
+	}
+	.subhead {
+		font-size: 1.625rem;
+	}
+
+/* ====== ABOUT ====== */
+.about {
+	margin-top: 65px;
+	padding: 0 4em;
+}
+	.about p {
+		font-size: 1.125rem;
+		line-height: 24px;
+		color: #333333;
+		font-weight: 400;
+		margin-bottom: 20px;
+	}
+	.about a {
+		font-weight: 600;
+		color: #2C3E50;
+		text-decoration: none;
+	}
+		.about a:hover {
+			text-decoration: underline;
+		}
+
+/* ====== CONTACT ====== */
+.contact {
```

Comment:
There are common traits between this and .about{} maybe think about using less unique classes that share similar properties. Like .page, then if something small changes you could refer to .page.contact{}